mypy/plugin.py

```diff
@@ -371,6 +371,19 @@ def lookup_fully_qualified(self, fullname: str) -> Optional[SymbolTableNode]:
         assert self._modules is not None
         return lookup_fully_qualified(fullname, self._modules)
 
+    def get_additional_deps(self, file: MypyFile) -> List[Tuple[int, str, int]]:
+        """Customize dependencies for a module.
+
+        This hook allows adding in new dependencies for a module. It is called
+        after parsing a file but before analysis.
+
+        Returns a list of (priority, module name, line number) tuples.
```

Comment:
Document that line number can be -1 if there's no specific line or the line is unknown.